src/components/api-provider.tsx

```diff
@@ -74,6 +74,12 @@ export type APIProviderProps = {
    * empty string. Read more in the
    * [documentation](https://developers.google.com/maps/reporting-and-monitoring/reporting#solutions-usage).
    */
+  channel?: string;
```

Comment:
If it's only numbers, should we maybe accept `string | number` as type here?

Reply:
@usefulthink Actually the urlParam expect key and value to be string because of which I changed the type to string BUT now I have got a solution for it so I changed the type back to number.